I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 23, in script
    $ password_list = [line.strip() for line in open("./john-the-ripper.txt", "r")]
  File "game/script.rpy", line 23, in <module>
    $ password_list = [line.strip() for line in open("./john-the-ripper.txt", "r")]
  File "game/script.rpy", line 23, in <lambda>
    $ password_list = [line.strip() for line in open("./john-the-ripper.txt", "r")]
IOError: [Errno 2] No such file or directory: u'./john-the-ripper.txt'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 23, in script
    $ password_list = [line.strip() for line in open("./john-the-ripper.txt", "r")]
  File "C:\Users\Farbod\renpy-7.5.3-sdk/renpy/ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Farbod\renpy-7.5.3-sdk/renpy/python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 23, in <module>
    $ password_list = [line.strip() for line in open("./john-the-ripper.txt", "r")]
  File "game/script.rpy", line 23, in <lambda>
    $ password_list = [line.strip() for line in open("./john-the-ripper.txt", "r")]
IOError: [Errno 2] No such file or directory: u'./john-the-ripper.txt'

Windows-10-10.0.22000 AMD64
Ren'Py 7.5.3.22090809
329Project 1.0
Fri Dec  2 17:01:49 2022
